com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -7,197 +7,13 @@
 
 namespace UnityEngine.Rendering.Universal
 {
+    /// <summary>
+    /// Deprecated, kept for backward compatibility with existing ForwardRendererData asset files.
+    /// Use StandardRendererData instead.
+    /// </summary>
+    [System.Obsolete("ForwardRendererData has been deprecated. Use StandardRendererData instead (UnityUpgradable) -> StandardRendererData", true)]
     [Serializable, ReloadGroup, ExcludeFromPreset]
-    [MovedFrom("UnityEngine.Rendering.LWRP")]
-    public class ForwardRendererData : ScriptableRendererData
+    public class ForwardRendererData : StandardRendererData
```

Comment:
I am not sure this will work. 
In general when you configure a type rename you need to keep all the public API in place but in this case you are changing the base type. I'd say you need to revert this (i.e, keep `ScriptableRendererData` as the base type and reintroduce the public members. 

also It looks like `ForwardRendererData` has been renamed to `StandardRendererData`. in this case the `MovedFrom` attribute need to stay in `ForwardRendererData` (not in `StandardRendererData`).

Reply:
Made `ForwardRendererData`  inherits from `ForwardRendererData` again, re-added the `MovedFrom` label, re-added the public members.